packages/chrome/src/styled/StyledSkipNav.ts

```diff
@@ -102,8 +110,8 @@ export const StyledSkipNav = styled.a.attrs({
 
   ${props => sizeStyles(props)};
 
-  &:focus {
-    outline: none;
+  ${SELECTOR_FOCUS_VISIBLE} {
+    text-decoration: none;
```

Comment:
This has a similar problem as my comment re: anchors, in general. A hover should display the underline.